src/mach/imports.rs

```diff
@@ -195,9 +216,9 @@ impl<'a> BindInterpreter<'a> {
 	            // throwBadBindingAddress(address, segmentEndAddress, segmentIndex, start, end, p);
 	            // (this->*handler)(context, address, type, symbolName, symboFlags, addend, libraryOrdinal, "", &last);
 	            // address += sizeof(intptr_t);
+                    imports.push(Import::new(&bind_info, libs, segments, start_of_sequence));
                     let seg_offset = bind_info.seg_offset.wrapping_add(ctx.size() as u64);
                     bind_info.seg_offset = seg_offset;
-                    imports.push(Import::new(&bind_info, libs, segments));
```

Comment:
Yea, but wouldn't this mean every single binding has substantially incorrect values; pointing to different libraries, different addresses, etc. I just feel like this would have shown up as very very broken, if it parsed at all.  Dunno.  I guess testing this against the canonical dyldinfo util on osx is just the correct way to check regressions, thanks for adding that!